OpenSim/Common/Test/testComponentInterface.cpp

```diff
@@ -1614,16 +1614,21 @@ void testTableReporter() {
             TimeStepper ts(system, integ);
             ts.initialize(s);
             ts.stepTo(1.0);
-            std::cout << "TableReporter table after simulating:\n";
             const auto& table = reporter->getTable();
-            SimTK_TEST_MUST_THROW_EXC(table.toString(), EmptyTable);
+            std::cout << "TableReporter table after simulating:\n"
+                      << table.toString() << std::endl;
+            SimTK_TEST_MUST_THROW_EXC(table.getDependentColumnAtIndex(0),
```

Comment:
> Would it be more informative to throw `EmptyTable` here?

Maybe, though it's unrecoverable so I'm not sure it matters too much. The fix is to add another `OPENSIM_THROW_IF()` to the top of [`DataTable_::getDependentColumnAtIndex()`](https://github.com/opensim-org/opensim-core/blob/master/OpenSim/Common/DataTable.h#L950) (and similar methods) to first throw if the table is empty (zero dependent columns and/or zero rows). You want me to do that?